backend/baton/src/main/java/touch/baton/domain/supporter/controller/response/SupporterResponse.java

```diff
@@ -19,7 +20,8 @@ public static Detail from(final Supporter supporter) {
                     supporter.getMember().getCompany().getValue(),
                     supporter.getReviewCount().getValue(),
                     supporter.getMember().getGithubUrl().getValue(),
-                    supporter.getIntroduction().getValue()
+                    supporter.getIntroduction().getValue(),
+                    supporter.getSupporterTechnicalTags().getSupporterTechnicalTags().toString()
```

Comment:
```suggestion
                    supporter.getSupporterTechnicalTags().getSupporterTechnicalTags().toString()
```

`toString()`을 사용하는 이유가 궁금해 🤔